plugin-server/src/ingestion/ingestion-consumer.test.ts

```diff
@@ -480,4 +492,201 @@ describe('IngestionConsumer', () => {
             expect(forSnapshot(getProducedKafkaMessages())).toMatchSnapshot()
         })
     })
+
+    describe('transformations', () => {
+        let transformationFunction: HogFunctionType
+        const TRANSFORMATION_TEST_TIMEOUT = 30000
+
+        beforeAll(() => {
+            jest.setTimeout(TRANSFORMATION_TEST_TIMEOUT)
```

Comment:
The fact this is necessary concerns me... With so much mocked out why do the tests need to run so long? Indicates something flakey or broken rather than just extending the timeout (as then we will never get to the bottom of the perf issue?)

Reply:
Is it because of the hog compilation when un-cached? Wondering if there is a way we can remove just that from the timeout calculations...

Reply:
On second thoughts lets move forward and fix later - only a comment would be good here to indicate for the future that this is due to the slow hog compliation